tests/Program.cs

```diff
@@ -0,0 +1,23 @@
+namespace TemporalioSamples.Tests;
+
+public static class Program
+{
+    internal static bool InProc { get; private set; }
+
+    internal static bool Verbose { get; private set; }
+
+    public static int Main(string[] args)
+    {
+        InProc = true;
+        Verbose = args.Contains("-verbose");
```

Comment:
Is it standard to use single dash flags in .NET?

Reply:
Seems inconsistent with `--help` below.

Reply:
This isn't my flag FWIW, it's just me wrapping XUnit's manual runner. I only wrapped it because the standard `dotnet test` runs a separate process that does not show live output well from CLI among other issues. Most won't use this.